spec/models/photo_uploader_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'carrierwave/test/matchers'
+
+describe PhotoUploader do
+  include CarrierWave::Test::Matchers
+
+  before do
+    PhotoUploader.enable_processing = true
+    @card = Card.new(user_id: 1)
+    @uploader = PhotoUploader.new(@card, :remote_photo_url)
+
+    File.open('spec/support/files/smile.jpg') do |f|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.